KPP/fullchem/commonIncludeVars.H

```diff
@@ -0,0 +1,214 @@
+  !==========================================================================
```

Comment:
New include file.  All of these variables (even if they aren't used in a mechanism) need to be present at compile time in order to avoid compilation errors.